ServiceNow/JFrog Spoke/README.md

```diff
@@ -65,7 +74,7 @@ Create a new API Key Credentials
 
 <img src="https://github.com/jfrog/partner-integrations/blob/main/ServiceNow/JFrog%20Spoke/images/m3.png?raw=true" width="800">
```

Comment:
If we are replacing absolute URL with relative ones, are we going to fix these too?